### PR TITLE
add a few unit tests for Pagination and fix some button styles

### PR DESCRIPTION
**Pull request by caradong**
#### Pull request checklist

- [ ] Addresses some steps in  #7254 
- [x] Include a change request file using `$ npm run change`

#### Description of changes
from Todo List in #7254 
Unit test (some UTs, not completed yet)
For the button display mode, remove the flashing effect of default button component
Make combo box the default option and reorder the examples accordingly
#### Focus areas to test
Pagination

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/7411)

pagination1214

### CODE REVIEW

File: packages/experiments/src/components/Pagination/examples/Pagination.Buttons.Customization.Round.Example.tsx [hunk 1]
```diff
@@ -49,8 +50,14 @@ export class PaginationCustomizationRoundExample extends React.Component<{}, IPa
                 borderRadius: '16px',
                 color: 'white'
               },
-              '&:hover:aria-selected=true': { color: 'white' },
-              '&:hover[aria-selected=true]': { color: 'white' }
+              '&:hover:aria-selected=true': {
```
Comment (KevinTCoughlin) [inline packages/experiments/src/components/Pagination/examples/Pagination.Buttons.Customization.Round.Example.tsx]: I believe you can remove this selector as it is `invalid` without the brackets which you've redefined below.

File: packages/experiments/src/components/Pagination/examples/Pagination.Buttons.Customization.Example.tsx [hunk 2]
```diff
@@ -35,12 +36,10 @@ export class PaginationCustomizationExample extends React.Component<{}, IPaginat
             selectors: {
               ':hover': { backgroundColor: '#c8c8c8', borderRadius: '16px' },
               '&:aria-selected=true': {
```
Comment (KevinTCoughlin) [inline packages/experiments/src/components/Pagination/examples/Pagination.Buttons.Customization.Example.tsx]: This selector is also _invalid_ (pretty sure) and redefined below. Should be safe to remove...

File: packages/experiments/src/components/Pagination/Pagination.styles.ts [hunk 3]
```diff
@@ -48,11 +48,26 @@ export function getStyles(props: IPaginationStyleProps): IPaginationStyles {
...
+            textDecoration: 'underline'
           },
           '&[aria-selected=true]': {
             color: palette.blue,
-            fontWeight: 'bold'
+            cursor: 'default',
+            fontWeight: 'bold',
+            textDecoration: 'underline'
+          },
+          '&:hover:aria-selected=true': {
```
Comment (KevinTCoughlin) [inline packages/experiments/src/components/Pagination/Pagination.styles.ts]: Same here... both of the above non-bracketed styles seem invalid or at least redundant. Should be able to safely remove.

File: packages/experiments/src/components/Pagination/Pagination.types.ts [hunk 4]
```diff
@@ -101,14 +101,14 @@ export interface IPaginationProps {
 
 export enum PaginationFormat {
```
Comment (KevinTCoughlin) [inline packages/experiments/src/components/Pagination/Pagination.types.ts]: Another way to define this which removes the need for an enum is to do 

```ts
export type PaginationFormat = 'comboBox' | buttons';
```

Which you might consider as part of this PR or a future PR.

Comment (micahgodbolt) [reply packages/experiments/src/components/Pagination/Pagination.types.ts]: yes, this is certainly a change we'd want made before major release. either in this PR or make note in the issue.

Comment (caradong) [reply packages/experiments/src/components/Pagination/Pagination.types.ts]: sure. Updated in this pr

---
In reply to: [242258511](https://github.com/OfficeDev/office-ui-fabric-react/pull/7411#discussion_r242258511) [](ancestors = 242258511)

File: packages/experiments/src/components/Pagination/examples/Pagination.Buttons.Customization.Round.Example.tsx (hunk 1 above)
Comment (caradong) [reply packages/experiments/src/components/Pagination/examples/Pagination.Buttons.Customization.Round.Example.tsx]: removed

---
In reply to: [241932197](https://github.com/OfficeDev/office-ui-fabric-react/pull/7411#discussion_r241932197) [](ancestors = 241932197)

File: packages/experiments/src/components/Pagination/examples/Pagination.Buttons.Customization.Example.tsx (hunk 2 above)
Comment (caradong) [reply packages/experiments/src/components/Pagination/examples/Pagination.Buttons.Customization.Example.tsx]: removed

---
In reply to: [241932229](https://github.com/OfficeDev/office-ui-fabric-react/pull/7411#discussion_r241932229) [](ancestors = 241932229)

File: packages/experiments/src/components/Pagination/Pagination.styles.ts (hunk 3 above)
Comment (caradong) [reply packages/experiments/src/components/Pagination/Pagination.styles.ts]: removed

---
In reply to: [241932268](https://github.com/OfficeDev/office-ui-fabric-react/pull/7411#discussion_r241932268) [](ancestors = 241932268)

File: packages/experiments/src/components/Pagination/Pagination.types.ts (hunk 4 above)
Comment (caradong) [reply packages/experiments/src/components/Pagination/Pagination.types.ts]: updated in this pr

---
In reply to: [241932420](https://github.com/OfficeDev/office-ui-fabric-react/pull/7411#discussion_r241932420) [](ancestors = 241932420)

Comment (KevinTCoughlin) [reply packages/experiments/src/components/Pagination/Pagination.types.ts]: Awesome, thank you 😃!

File: packages/experiments/src/components/Pagination/Pagination.styles.ts (hunk 3 above)
Comment (KevinTCoughlin) [reply packages/experiments/src/components/Pagination/Pagination.styles.ts]: Thank you 👍

**Comment by KevinTCoughlin** [comment]
@caradong looks like screenshots need to be re-generated, then this can be merged.

https://travis-ci.org/OfficeDev/office-ui-fabric-react/builds/469182051#L2899

**Comment by caradong** [comment]
My bad. Updated now.

---
In reply to: [447988193](https://github.com/OfficeDev/office-ui-fabric-react/pull/7411#issuecomment-447988193) [](ancestors = 447988193)

**Comment by KevinTCoughlin** [comment]
> 
> 
> My bad. Updated now.
> 
> In reply to: [447988193](https://github.com/OfficeDev/office-ui-fabric-react/pull/7411#issuecomment-447988193) [](ancestors = 447988193)

No worries at all, thank you for pushing that up! @micahgodbolt are we good to merge once green?

**Pull request merged**